addons/web/static/src/js/tools/test_menus.js

```diff
@@ -203,7 +205,7 @@
     function testFilters() {
         var filterProm = Promise.resolve();
         // var $filters = $('div.o_control_panel div.btn-group.o_dropdown > ul.o_filters_menu > li:not(.o_add_custom_filter)');
-        var $filters = $('.o_filters_menu > .o_menu_item');
+        var $filters = $('.o_filters_menu > .o_menu_item:not(.d-none)');
```

Comment:
```js
        var $filters = $('.o_filters_menu > .o_menu_item:visible');
```

Reply:
as we discussed, the clickEverywhere test does not open the dropdown, so the filter is not visible. I keep the 'd-none' solution.